2019-06-14 02:03:10,600 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 02:03:37,911 ERROR: Exception on /edit_profile [POST] [in /home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/flask/app.py:1780]
Traceback (most recent call last):
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1244, in _execute_context
    cursor, statement, parameters, context
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 550, in do_execute
    cursor.execute(statement, parameters)
sqlite3.IntegrityError: UNIQUE constraint failed: user.username

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/app/routes.py", line 78, in edit_profile
    db.session.commit()
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/scoping.py", line 162, in do
    return getattr(self.registry(), name)(*args, **kwargs)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 1026, in commit
    self.transaction.commit()
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 493, in commit
    self._prepare_impl()
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 472, in _prepare_impl
    self.session.flush()
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 2458, in flush
    self._flush(objects)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 2596, in _flush
    transaction.rollback(_capture_exception=True)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/util/langhelpers.py", line 68, in __exit__
    compat.reraise(exc_type, exc_value, exc_tb)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 129, in reraise
    raise value
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 2556, in _flush
    flush_context.execute()
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/unitofwork.py", line 422, in execute
    rec.execute(self)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/unitofwork.py", line 589, in execute
    uow,
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/persistence.py", line 236, in save_obj
    update,
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/orm/persistence.py", line 978, in _emit_update_statements
    statement, multiparams
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 988, in execute
    return meth(self, multiparams, params)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1107, in _execute_clauseelement
    distilled_params,
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1248, in _execute_context
    e, statement, parameters, cursor, context
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1466, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 383, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 128, in reraise
    raise value.with_traceback(tb)
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1244, in _execute_context
    cursor, statement, parameters, context
  File "/home/wedeueis/Documentos/Programação/Flask/microblog/venv/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 550, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityError) UNIQUE constraint failed: user.username
[SQL: UPDATE user SET username=?, about_me=? WHERE user.id = ?]
[parameters: ('susan', "I'm the site developer.", 2)]
(Background on this error at: http://sqlalche.me/e/gkpj)
2019-06-14 02:12:32,680 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 13:12:40,800 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 13:13:02,274 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 16:01:43,598 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 16:02:22,288 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 16:04:20,540 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 16:22:06,093 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 16:45:48,998 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 16:46:13,559 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 17:00:45,121 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 17:05:49,110 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 17:24:43,971 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 17:37:17,967 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 17:50:10,252 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 17:58:15,239 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
2019-06-14 17:58:18,261 INFO: Microblog startup [in /home/wedeueis/Documentos/Programação/Flask/microblog/app/__init__.py:44]
